lib/ovirt_metrics.rb

```diff
@@ -59,7 +59,8 @@ def self.host_realtime(host_id, start_time = nil, end_time = nil)
     host_realtime_metrics_to_hashes(metrics, nic_metrics)
   end
 
-  private
+  # The methods below this line are PRIVATE not meant to be used out of the scope
```

Comment:
I changed this due to a Rubocop message. THe private modifier here was actually doing nothing because it does not work this way for class methods. So I replaced it with this comment.